_data/header.yml

```diff
@@ -3,12 +3,21 @@
 # Uncomment this statement (by removing the leading '# ') to set the
 # header title differently from the site's title (as defined in
 # _config.yml).
-# title: Header title
+
+title: ""
+
+# Used to show or hide the official usa banner at the top of the site.
+# Comment out to the following line to hide the usa banner.
+usa_banner: true
+
+# this defines the type of header
+# header types can be 'basic' or 'extended'
```

Comment:
maybe a succinct explanation of `basic` vs `extended` would be nice

Reply:
Do you think URLs to the component library would be helpful? 
https://components.designsystem.digital.gov/components/detail/header--basic.html
https://components.designsystem.digital.gov/components/detail/header--extended.html

Reply:
oh, that's a good idea!